packages/components/src/DataTable/DataTable.mdx

```diff
@@ -0,0 +1,224 @@
+---
+name: DataTable
+menu: Patterns
+route: /components/data-table
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { DataTable } from ".";
+
+# Data Table
+
+```ts
+import { DataTable } from "@jobber/components/DataTable";
+```
+
+<Playground>
+  <DataTable
+    data={[
+      {
+        name: "Eddard",
+        house: "Stark",
+        region: "North",
+        sigil: "Direwolf",
+        isAlive: "No",
+      },
+      {
+        name: "Catelyn",
+        house: "Stark",
+        region: "North",
+        sigil: "Direwolf",
+        isAlive: "No",
+      },
+      {
+        name: "Jon Snow",
+        house: "Stark",
+        region: "North",
+        sigil: "Direwolf",
+        isAlive: "Yes",
+      },
+      {
+        name: "Robert",
+        house: "Stark",
+        region: "North",
+        sigil: "Direwolf",
+        isAlive: "No",
+      },
+      {
+        name: "Rickon",
+        house: "Stark",
+        region: "North",
+        sigil: "Direwolf",
+        isAlive: "No",
+      },
+      {
+        name: "Robert",
+        house: "Baratheon",
+        region: "Stormlands",
+        sigil: "Black Stag",
+        isAlive: "No",
+      },
+      {
+        name: "Cercei",
+        house: "Lannister",
+        region: "Westerlands",
+        sigil: "Golden Lion",
+        isAlive: "Yes",
+      },
+      {
+        name: "Sansa",
+        house: "Stark",
+        region: "North",
+        sigil: "Direwolf",
+        isAlive: "Yes",
+      },
+      {
+        name: "Arya",
+        house: "Stark",
+        region: "North",
+        sigil: "Direwolf",
+        isAlive: "Yes",
+      },
+      {
+        name: "Bran",
+        house: "Stark",
+        region: "North",
+        sigil: "Direwolf",
+        isAlive: "Yes",
+      },
+      {
+        name: "Joffrey",
+        house: "Baratheon",
+        region: "Stormlands",
+        sigil: "Black Stag",
+        isAlive: "No",
+      },
+      {
+        name: "Myrcella",
+        house: "Baratheon",
+        region: "Stormlands",
+        sigil: "Black Stag",
+        isAlive: "Yes",
+      },
+      {
+        name: "Tommen",
+        house: "Baratheon",
+        region: "Stormlands",
+        sigil: "Black Stag",
+        isAlive: "Yes",
+      },
+    ]}
+    pagination={{ manualPagination: false, itemsPerPage: [10, 20, 30] }}
+    sorting={{ manualSorting: false }}
+    columns={[
```

Comment:
I know several people had concerns over what kind of data that could be shown in a table, so it would also be cool to have an example where Icon/Avatar are used in a column

Reply:
We definitely should update the examples to have more use cases. We just hadn't had the time yet tough. We are trying to get the features working first, and go back to make it better. Right now we are using the playground literally as a playground but we should soon revisit it in order to update the props descriptions and examples.

Reply:
Yeah, that's reasonable. I'm fine with that being a follow up if getting this in a better state ends up being blocking 👍